src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString.split(';').forEach((declaration) => {
+    let [property, value] = declaration.split(':');
+
+    if (property && value) {
+      property = property.trim();
+      value = value.trim();
+      styles[property] = value;
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. While you are using `forEach` which is an iteration method, it is generally used for side effects. A method like `reduce` would be more suitable here as it can transform an array into an object without side effects.